.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+repos:
+-   repo: https://github.com/ambv/black
+    rev: 19.3b0
+    hooks:
+    -   id: black
+        args: [--safe, --quiet]
+        language_version: python3
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: debug-statements
+    -   id: flake8
```

Comment:
@nicoddemus What are your thoughts about `flake8`?

I think that we can drop it from pre-commit (recently added) and from the Tox also (already in place).

Reply:
I agree we can drop it. 👍

Reply:
Okay, I will remove `flake8` then. The formatter is enough, in my opinion.